# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/locales/en-US/chrome/dom/dom.properties
# Commit: 3cd097a7b017
# Full Hash: 3cd097a7b017154cf5f7a2d9ed97c7891d3074f4
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-10 09:58:56
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=baku,flod
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D26654
# ==============================================================================

diff -r ea2baf2c9ae7 -r 3cd097a7b017 dom/locales/en-US/chrome/dom/dom.properties
--- a/dom/locales/en-US/chrome/dom/dom.properties	Tue Apr 09 20:32:54 2019 +0000
+++ b/dom/locales/en-US/chrome/dom/dom.properties	Wed Apr 10 00:14:39 2019 +0000
@@ -358,6 +358,8 @@
 IDBOpenDBOptions_StorageTypeWarning=The ‘storage’ attribute in options passed to indexedDB.open is deprecated and will soon be removed. To get persistent storage, please use navigator.storage.persist() instead.
 DOMQuadBoundsAttrWarning=DOMQuad.bounds is deprecated in favor of DOMQuad.getBounds()
 UnsupportedEntryTypesIgnored=Ignoring unsupported entryTypes: %S.
+# LOCALIZATION NOTE: Do not add a period at the end of this sentence. This string is used in UnsupportedEntryTypesIgnored, it will be preceded by a colon and followed by a period.
+NoValidEntryTypes=No valid entry types; aborting registration
 
 #LOCALIZATION NOTE(DeprecatedTestingInterfaceWarning): Do not translate this message. It's just testing only.
 DeprecatedTestingInterfaceWarning=TestingDeprecatedInterface is a testing-only interface and this is its testing deprecation message.